<?xml version="1.0"?>
<robot name="victor_with_rod" xmlns:xacro="http://www.ros.org/wiki/xacro">
    <xacro:include filename="$(find gpu_voxel_planning)/urdf/victor_right_arm_only.urdf.xacro"/>


    <!-- Pitcher -->
    <link name="rod">
      <collision>
        <origin xyz="0 0 -0.02" rpy="0 0 0"/>
        <geometry>
          <cylinder length="0.234" radius="0.065"/>
        </geometry>
      </collision>
    </link>
    <joint name="rod_joint" type="fixed">
      <parent link="victor_right_gripper_palm"/>
      <child link="rod"/>
      <origin xyz="0 0 0.15" rpy="1.57 0 0"/>
      <axis xyz="0 0 1"/>
    </joint>

    <!-- Pringles -->
    <!-- <link name="rod"> -->
    <!--   <collision> -->
    <!--     <origin xyz="0 0 0.30" rpy="0 0 0"/> -->
    <!--     <geometry> -->
    <!--       <cylinder length="0.234" radius="0.038"/> -->
    <!--     </geometry> -->
    <!--   </collision> -->
    <!-- </link> -->
    <!-- <joint name="rod_joint" type="fixed"> -->
    <!--   <parent link="victor_right_gripper_palm"/> -->
    <!--   <child link="rod"/> -->
    <!--   <origin xyz="0 0 0.0" rpy="0 0 0"/> -->
    <!--   <axis xyz="0 0 1"/> -->
    <!-- </joint> -->
</robot>
